evm_arithmetization/src/witness/operation.rs

```diff
@@ -1,8 +1,12 @@
+use std::collections::HashMap;
+use std::hash::RandomState;
```

Comment:
`RandomState` is the default for HashMaps, it can be elided everywhere it's mentioned.